chain/client/Cargo.toml

```diff
@@ -42,10 +42,10 @@ near-performance-metrics-macros = { path = "../../utils/near-performance-metrics
 near-vm-runner = { path = "../../runtime/near-vm-runner" }
 delay-detector = { path = "../../tools/delay_detector", optional = true }
 near-network-primitives = { path = "../network-primitives" }
+near-actix-test-utils = { path = "../../test-utils/actix-test-utils" }
+near-logger-utils = { path = "../../test-utils/logger" }
```

Comment:
Do we need to move these dependencies? I think we shouldn't because dev-dependencies should work for unit tests

Reply:
@mzhangmzz Thanks, fixed.